### PR TITLE
#160550290 heroku host

### PR DESCRIPTION
**Pull request by nwashangai**
#### What does this PR do?
It hosts the API on Heroku
#### Description of Task to be completed?
* Include babel CLI on package.json file
#### How should this be manually tested?
* Not applicable
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
#160550290
#### Screenshots (if appropriate)
* None
#### Questions:
* None

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -19,8 +19,6 @@ app.use((req, res, next) => {
   res.send({ error: '404 Sorry the page has not yet been defined try /api/v1/' });
 });
 
-app.listen(process.env.PORT || 3000, () => {
-  console.log('server started at port 3000');
-});
+app.listen(process.env.PORT || 3000, () => {});
 
 module.exports = app; // for testing
```
Comment (dogoyaro) [inline src/app.js]: Take out the comment. I honestly don't think it is needed.

File: package.json [hunk 2]
```diff
@@ -4,13 +4,14 @@
   "private": true,
   "main": "app.js",
   "scripts": {
-    "start": "babel-node ./src/app.js",
-    "dev": "babel-node ./src/app.js",
+    "start": "babel-node src/app.js",
```
Comment (dogoyaro) [inline package.json]: babel-node is not used in production. We'll use this now, but note that you have to change this before starting challenge 3.

**Review by dogoyaro — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (nwashangai) [reply package.json]: Ok noted

**Comment by dogoyaro** [comment]
You can go ahead to merge this.

**Pull request merged**